src/Gui/InputHintWidget.cpp

```diff
@@ -234,27 +234,41 @@ QString Gui::InputHintWidget::inputRepresentation(const InputHint::UserInput key
         case KeyAsciiTilde: return QStringLiteral("~");
 
         // misc keys
-        case KeyEscape: return tr("Escape");
-        case KeyTab: return tr("tab ⭾");
+        /*: Keyboard key for Escape */
+        case KeyEscape: return tr("Esc");
```

Comment:
```suggestion
        case KeyEscape:
                /*: Keyboard key for Escape */
                return tr("Esc");
```

You might have already checked this, so please ignore this suggestion if you have and the translator comment is correctly extracted and associated to the string. I'm not sure how immediately before the `tr()` call the translator comment needs to be.

Reply:
Both should work, keeping as is to keep readability